pkg/build/builder/docker.go

```diff
@@ -47,6 +48,17 @@ type DockerBuilder struct {
 	urlTimeout   time.Duration
 }
 
+// MetaInstuction represent an Docker instruction used for adding metadata
+// to Dockerfile
+type MetaInstruction string
+
+const (
+	// Label represents the LABEL Docker instruction
+	Label MetaInstruction = "LABEL"
```

Comment:
is there a reason to have this as constant? i'm 99% sure it wont change anytime soon